apps/meteor/app/lib/server/functions/cleanRoomHistory.ts

```diff
@@ -86,7 +86,7 @@ export async function cleanRoomHistory({
 		}
 	}
 
-	const count = await Messages.removeByIdPinnedTimestampLimitAndUsers(
+	const { count, selectedMessageIds } = await Messages.removeByIdPinnedTimestampLimitAndUsers(
```

Comment:
I believe this could lead to confusion, since `selectedMessageIds` will be undefined if limit is undefined or 0. We are indeed checking it below, but I think it should be more explicit, I'd say it would be better to separate the model method into 2, one for deleting and another one for querying. What is your opinion?